### PR TITLE
BasePicker: Fix accessibility bug where it always has aria-expanded=true

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

BasePicker: Fix accessibility bug where it always has aria-expanded=true

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx [hunk 1]
```diff
@@ -199,7 +199,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                 suggestedDisplayValue={ suggestedDisplayValue }
                 aria-activedescendant={ 'sug-' + this.suggestionStore.currentIndex }
                 aria-owns='suggestion-list'
-                aria-expanded='true'
+                aria-expanded={ this.state.suggestionsVisible ? 'true' : 'false' }
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: just return this wrapped with Boolean(asdf)

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
       // If there are less new items than old items then something was removed and we
       // should try to keep focus consistent
       if (newItems.length < this.state.items.length) {
-        focusIndex = this.state.items.indexOf(this.selection.getSelection()[0]);
+        focusIndex = this.state.items.indexOf(this.selection.getSelection() [0]);
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: ?

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx [hunk 3]
```diff
@@ -272,7 +272,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     let { items } = this.state;
 
     if (items.length && index! >= 0) {
-      let newEl: HTMLElement = this.root.querySelectorAll('[data-selection-index]')[Math.min(index!, items.length - 1)] as HTMLElement;
+      let newEl: HTMLElement = this.root.querySelectorAll('[data-selection-index]') [Math.min(index!, items.length - 1)] as HTMLElement;
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: formatting changes doen't seem right

Comment (joschect) [reply packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: That was automatic. I didn't want to do it. I'll change it.

**Pull request merged**